scripts/get-agent.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs')
```

Comment:
This file is almost identical to `scripts/prepareHoloFuelData/agent-config.js`.  Can we reference this file or somehow merge these, so avoid duplicate code?